### ISSUE TITLE
(dev2.0) CLI alpha-031 can't detect Fable.Core version and fails to compile a project

### ISSUE THREAD

**Issue by Zaid-Ajaj** [fable-compiler/Fable#1495]
### Description

Updating [Fable.SimpleJson](https://github.com/Zaid-Ajaj/Fable.SimpleJson) to CLI alpha-031 doesn't compile the project because it can't detect Fable.Core version 

### Repro code

- Fresh clone SimpleJson 
- Run the tests (they pass)
- Update Fable cli 
- Run the tests again 
- See the error:
```
ERROR in ./test/Tests.fsproj
Module build failed: Error: Fable.Core v0.0.0 detected, expecting v2.0.0
    at Loader.fableUtils.client.send.then.r (C:\projects\Fable.SimpleJson\node_modules\fable-loader\index.js:67:22)
```
Concrete steps
```bash
# Clone repo
git clone https://github.com/Zaid-Ajaj/Fable.SimpleJson.git
cd Fable.SimpleJson 
# Run the tests
./build.sh RunTests
# They should pass
(mono) .paket/paket.exe update dotnet-fable --version 2.0.0-alpha-031 
# Run the tests again 
./build.sh RunTests
# See the error message
``` 
### Related information

* Fable version (`dotnet fable --version`): 2.0.0-alpha-031
* Operating system: Windows 10

**Comment by alfonsogarciacaro** [comment]
I think this is because I didn't include Assembly Info in previous Fable.Core 2 alpha releases. Can you please try with latest one 2.0.0-alpha-014?

**Comment by Zaid-Ajaj** [comment]
OK I am confused, now it is working correctly 😅 this is what I did
- git clone https://github.com/Zaid-Ajaj/Fable.SimpleJson.git 
- cd Fable.SimpleJson
- delete paket.lock 
- inside paket.dependencies, change `dotnet-fable ~> 2.0.0-alpha-019`  to `dotnet-fable >= 2.0.0-alpha-031` 
 - ./build.sh RunTests 

(resolved dotnet-fable 2.0.0-alpha-031 and Fable.Core 2.0.0-alpha-014)

maybe it was something with my local `.nuget` cache?

**Comment by alfonsogarciacaro** [comment]
Nah, it's fine. I guess when dotnet-fable CLI was updated, Fable.Core remained 2.0.0-alpha-012 (or previous) :)

I'm closing the issue for now but please let me know if you've questions.
